server/controllers/stock/core.js

```diff
@@ -469,7 +469,7 @@ function getInventoryQuantityAndConsumption(params) {
       d.text AS depot_text, l.unit_cost, l.expiration_date,
       ROUND(DATEDIFF(l.expiration_date, CURRENT_DATE()) / 30.5) AS lifetime,
       BUID(l.inventory_uuid) AS inventory_uuid, BUID(l.origin_uuid) AS origin_uuid,
-      l.entry_date, i.code, i.text, BUID(m.depot_uuid) AS depot_uuid,
+      l.entry_date, BUID(i.uuid) AS inventory_uuid, i.code, i.text, BUID(m.depot_uuid) AS depot_uuid,
```

Comment:
It looks like this property already exists in this SQL statement on the line directly above it:  `BUID(l.inventory_uuid) AS inventory_uuid`.  Did that not work?

Reply:
What is the difference between `BUID(l.inventory_uuid) AS inventory_uuid` in the line above and `BUID(i.uuid) AS inventory_uuid` in this line? 

I worry we've changed the behavior of this SQL and that will have negative impacts somewhere.